src/Make_mvc.mak

```diff
@@ -1446,6 +1446,13 @@ clean: testclean
 cmdidxs: ex_cmds.h
 	vim --clean -X --not-a-term -u create_cmdidxs.vim
 
+# Run vim script to generate the normal/visual mode command lookup table.
+# This only needs to be run when a new normal/visual mode command has been
+# added.  If this fails because you don't have Vim yet, first build and
+# install Vim without changes.
+nvcmdidxs: nv_cmdidxs.h
+	vim --clean -X --not-a-term -u create_nvcmdidxs.vim
```

Comment:
This `vim` must be what is built from the "current" sources, unlike of `make cmdidxs`, mustn't it?
Namely, I feel, needs the these steps when update `nv_cmdidxs.h`:

1. change `nv_cmds[]` in normal.c
2. build vim
3. do `make nvcmdidxs` using vim built in step 2
4. rebuild vim (if need)

Reply:
Yes. I will update the comments to reflect that.

Reply:
Should use `./$(VIMTARGET)` instead of `vim`.

Reply:
Thanks for the suggestion. I have updated the PR.